v21.2/crdb-internal.md

```diff
@@ -5,128 +5,638 @@ toc: true
+ORDER BY total_access desc;
 ~~~
 
+##### Which indexes are infrequently used? Are there any unused indexes?
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SELECT  ti.descriptor_name as table_name, ti.index_name, total_reads
+FROM crdb_internal.index_usage_statistics AS us
+JOIN crdb_internal.table_indexes ti
+ON us.index_id = ti.index_id AND us.table_id = ti.descriptor_id
+WHERE total_reads = 0;
+~~~
+
+### `statement_statistics`
+
+Column | Type | Description
+------------|-----|------------
+`aggregated_ts` | `TIMESTAMPTZ NOT NULL` | The time that statistics aggregation started.
+`fingerprint_id` | `BYTES NOT NULL` | Unique identifier of the statement statistics. This is constructed using the statement fingerprint text, and statement metadata (e.g. query type, database name, etc.)
+`transaction_fingerprint_id` | `BYTES NOT NULL` | Uniquely identifies a transaction statistics. The transaction fingerprint ID that this statement statistic belongs to.
+`plan_hash` | `BYTES NOT NULL` | Uniquely identifies a query plan that was executed by the current statement. The query plan can be retrieved from the `sampled_plan` column.
+`app_name` | `STRING NOT NULL`| The name of the application that executed the statement.
+`metadata` | `JSONB NOT NULL` | Metadata that describes the statement. See [`metadata` column](#metadata-column).
+`statistics` | `JSONB NOT NULL` | Statistics for the statement. See [`statistics` column](#statistics-column).
+`sampled_plan` | `JSONB NOT NULL` | The sampled query plan of the current statement statistics. This column is unfilled if there is no sampled query plan.
+`aggregation_interval` | `INTERVAL NOT NULL` | The interval over which statistics are aggregated.
+
+### `metadata` column
+
+Field | Type | Description
+------------|-----|------------
+`db` | `STRING` | The database on which the statement is executed.
+`distsql` | `BOOLEAN` | Whether the statement is being executed by the Distributed SQL (DistSQL) engine.
+`failed` | `BOOLEAN` | Whether the statement execution failed.
+`fullScan` | `BOOLEAN` | Whether the statement performed a full scan of the table.
+`implicitTxn` | `BOOLEAN` | Whether the statement executed in an implicit transaction.
+`query` | `STRING` | The statement string.
+`querySummary` | `STRING` | The statement string summary.
+`stmtTyp` | `SQLType` | The type of statement: `TypeDDL`, `TypeDML`, `TypeDCL`, or `TypeTCL`.
+`vec` | `BOOLEAN` | Whether the statement executed in the vectorized query engine.
+
+### `statistics` column
+
+`execution_statistics` are the execution statistics sent to the DB Console for a statement fingerprint.
+`statistics` are the statement statistics sent to the DB Console for a statement fingerprint.
+
+The `NumericStat` type tracks two running values: the running mean `mean` and the running sum of squared differences `sqDiff` from the mean. Using these along with the total count of values, CockroachDB computes the variance using Welford's method.
```

Comment:
Where does CockroachDB compute and surface this variance? @Azhng 

Should we just remove the text starting "Using these..." Or call out "you can compute the variance using Welford's method"?

Reply:
I wondered myself when I was writing that. Is the user interested in the `sqDiff` or the variance?

Reply:
We render the [`barChart`](https://github.com/cockroachdb/cockroach/blob/94c71f0be58e48f557a08165368acff51b7ee722/pkg/ui/workspaces/cluster-ui/src/statementsTable/statementsTable.tsx#L72) in the Statement Page and Transaction Page using the variance / stddev derived from this `sqDiff` value. 

That tiny grey bar (sometimes there is a blue line around it) is what I referred to as `barChart` 

<img width="128" alt="Screen Shot 2022-02-04 at 16 14 54" src="https://user-images.githubusercontent.com/9267198/152604038-e68538dc-d719-4420-b9b8-8099f04914ef.png">